workshop_app/send_mails.py

```diff
@@ -148,11 +148,21 @@ def send_email(	request, call_on,
 						In case of queries regarding workshop booking(s), 
 						revert to this email.""".format(PRODUCTION_URL, key))
 
+			message_to_admin = dedent("""\
+								A new coordinator has registered.
+								coordinator email: {0}
+								""".format(request.user.email))
+
 			try:
 				send_mail(
 					"Coordinator Registration at FOSSEE, IIT Bombay", message, SENDER_EMAIL, 
-					[request.user.email, 'workshops@fossee.in'], fail_silently=False
+					[request.user.email], fail_silently=False
 					)
+
+				send_mail("Coordinator Registration at FOSSEE, IIT Bombay", message_to_admin,
+							SENDER_EMAIL, ['workshops@fossee.in'], fail_silently=False
```

Comment:
Do we really want to bother admin(s)? If yes, should then admin be aware of all the thousand workshops happening over a course of short period?